lib/pf/pfcmd/checkup.pm

```diff
@@ -1296,6 +1297,18 @@ sub cert_has_expired {
     return time > $expiration;
 }
 
+=item advanced_config
+
+Check the advanced config section
+
+=cut
+
+sub advanced_config {
```

Comment:
wouldn't it be better to check that all numeric fields defining a minimum are respecting it.

Would be less of a case by case and make it cleaner